cmd/percona-dbaas/psmdb/upgrade.go

```diff
@@ -32,7 +32,7 @@ import (
 // upgradeCmd represents the edit command
 var upgradeCmd = &cobra.Command{
 	Use:   "upgrade <psmdb-cluster-name> <to-version>",
-	Short: "Upgrade MySQL cluster",
+	Short: "Upgrade Percona Server MongoDB cluster",
```

Comment:
```suggestion
	Short: "Upgrade Percona Server for MongoDB",
```
according to https://www.percona.com/software/mongo-database/percona-server-for-mongodb

Reply:
done